app/views/user_mailer/invitation_to_claim.text.erb

```diff
@@ -1,34 +1,33 @@
 <% content_for :message do %>
-  Hello from the Archive of Our Own!
+  <%= t '.greeting', app_name: ArchiveConfig.APP_NAME %>
 
-  You're receiving this e-mail because an archive has recently been imported by Open Doors (http://opendoors.transformativeworks.org) into the Archive of Our Own (AO3 - <%= root_url %>), and we believe that the following fanworks belong to you. We'd like to give you a chance to claim (or delete/orphan) these works if you want to. And if you don't already have an account under a different e-mail, we'd like to invite you aboard!
+  <%= t '.text.part1', open_doors_link: "http://opendoors.transformativeworks.org", app_name: ArchiveConfig.APP_NAME, app_short_name: ArchiveConfig.APP_SHORT_NAME, app_url: root_url %>
 
-  You can read announcements about recent archive moves at AO3 News (<%= root_url %>admin_posts?tag=18), and find additional information on Open Doors' FAQ page (http://opendoors.transformativeworks.org/faq) or tutorials page (http://opendoors.transformativeworks.org/tutorials). For any questions not answered in the FAQ, tutorials, or this e-mail, please contact Support at <%= root_url %>support.
+  <%= t '.text.part2', news_link: admin_posts_url + "?tag=18", open_doors_faq_link: "http://opendoors.transformativeworks.org/faq", open_doors_tutorial_link: "http://opendoors.transformativeworks.org/tutorials", support_link: new_feedback_report_url %>
 
-  If this is a mistake and these are not your works, please don't delete them! Please just contact Open Doors (http://transformativeworks.org/contact/open%20doors) and we will sort it out.
+  <%= t '.text.part3', open_doors_link: "http://transformativeworks.org/contact/open%20doors" %>
 
-  Depending on the archive, your works may have been imported restricted to registered users only (to keep them out of Google searches). If this is the case, the works will only be accessible by logged-in users unless you choose to make them fully visible. For help unlocking, orphaning, or deleting your works, please contact AO3 Support.
+  <%= t '.text.part4' %>
 
-  Claim or remove your works: <%= claim_url(:invitation_token => @token) %>
+  <%= t '.part5' %>: <%= claim_url(:invitation_token => @token) %>
 
-  The works uploaded include:
+  <%= t '.part6' %>:
```

Comment:
It looks like we include the colon in the translatable string for other emails (e.g. `    challenge_assignment_notification`)

Reply:
It had the colon in the text but not the HTML version, hence not combining it, but that looks like just a mistake so I've added it in.